dts/arm/infineon/psoc6/psoc6_04/psoc6_04.dtsi

```diff
@@ -36,12 +36,6 @@
 			write-block-size = <512>;
 			erase-block-size = <512>;
 		};
-		flash1: flash@14000000 {
```

Comment:
This might break some other use cases. Is there a way to push this node to a separate dtsi?

Reply:
I can delete it in the board device tree.  The size for this flash is defined as 0 so I was getting an error. It also does not exist according to the vendor docs so that's why I deleted it here. But  I'm fine to delete it in board specific dts!